.github/workflows/ecr-docker-build.yml

```diff
@@ -0,0 +1,50 @@
+name: Docker build and push to ECR
```

Comment:
We should probably chance the file name and the workflow name now its pushing the template as well.... maybe `post-merge.yml`?